goal_src/jak2/pc/features/autosplit-h.gc

```diff
@@ -0,0 +1,148 @@
+;;-*-Lisp-*-
+(in-package goal)
+
+;; LiveSplit ASL requires all settings to initalized _before_ you connect the process
+;; Therefore everything has to be laid out in a predictable fashion before hand
+;; So this is a lot of hard-coding, but not too bad when just copied from the debug menu code
+;;
+;; DO NOT change the order, appending to the end is safe!
+;;Hacks here
+
+
+;;End Hacks
+
+(deftype autosplit-info-jak2 (structure)
+  (;; General stats
+   (num-power-cells uint32)
```

Comment:
something i should have did in jak 1 from the get-go -- let's append 2 version numbers (major and minor) to the beginning of the struct, two `uint16`s.  This will let us make breaking changes / roll out changes conditionally instead of having coupled releases between jak-project and the autosplitter script.